From 18515831c3efac739264591ef8cca8de807d4954 Mon Sep 17 00:00:00 2001
From: Ulises Cardenas <Ulises.Cardenas@freescale.com>
Date: Fri, 12 Jun 2015 05:54:36 -0500
Subject: [PATCH 0905/5242] MLK-11103 Missing register in Secure memory
 configuration v1

commit  e5a6593eb4273b99c70144a9c3409bb94bcdcfb5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Added missing reserved register for caam_secure_mem_v1 struct

Signed-off-by: Dan Douglass <dan.douglass@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/crypto/caam/regs.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/crypto/caam/regs.h b/drivers/crypto/caam/regs.h
index 02d1ad7..03a38d6 100644
--- a/drivers/crypto/caam/regs.h
+++ b/drivers/crypto/caam/regs.h
@@ -612,6 +612,7 @@ struct caam_secure_mem_v1 {
 	u32 sm_cmd;	/* SMCJRx - Secure memory command */
 	u32 rsvd1;
 	u32 sm_status;	/* SMCSJRx - Secure memory status */
+    u32 rsvd2;
 
 	u32 sm_perm;	/* SMAPJRx - Secure memory access perms */
 	u32 sm_group2;	/* SMAP2JRx - Secure memory access group 2 */
-- 
1.7.9.5

